project-base/storefront/public/locales/cs/common.json

```diff
@@ -141,6 +142,10 @@
     "Go to cart": "Přejít do košíku",
     "Great choice! We have added your item to the cart": "Skvělá volba! Vaše zboží jsme přidali do košíku",
     "Hide results": "Skrýt výsledky",
+    "Hide the next {{ count }} best sellers_one": "Skrýt {{ count }} následující produkt",
+    "Hide the next {{ count }} best sellers_few": "Skrýt {{ count }} následující produkty",
+    "Hide the next {{ count }} best sellers_many": "Skrýt {{ count }} následujících produktů",
+    "Hide the next {{ count }} best sellers_other": "Skrýt {{ count }} následujících produktů",
```

Comment:
This seems to me very overwhelming. If it is not required by business, I don't see any beneficial here. Just bunch of additional unnecessary data on client. If you agree let's put just simple "Show more"/"Show less" as suggested in previous comment.

Reply:
Yeah, TBH I agree with you :laughing: